### PR TITLE
MWPW-156125- Pass ietf code for locale mapping at prelude side

### PR DESCRIPTION
**Pull request by Ruchika4**
- Pass ietf code for locale mapping at prelude side

Resolves: [MWPW-156125](https://jira.corp.adobe.com/browse/MWPW-156125)

Test URLs:

Before: https://main--cc--adobecom.hlx.page/drafts/ruchika/remove-background1?unitylibs=stage
After: https://main--cc--adobecom.hlx.page/drafts/ruchika/remove-background1?unitylibs=locale

Note: Code update has been done for both V2 and V1

### CODE REVIEW

File: unitylibs/scripts/utils.js [hunk 1]
```diff
@@ -64,6 +64,14 @@ export function defineDeviceByScreenSize() {
   return 'TABLET';
 }
 
+export function getLocale() {
+  const urlLocale = window.location.pathname.split('/')[1];
+  const config = getConfig();
+  const { locales } = config;
```
Comment (robert-bogos) [inline unitylibs/scripts/utils.js:70]: Nit: this can be simplified
```suggestion
  const { locales } = getConfig();
```

**Review by robert-bogos — APPROVED** [review]

**Review by robert-bogos — COMMENTED** [review]
The reset button looks broken, is that expected? 

image

**Comment by Ruchika4** [comment]
> The reset button looks broken, is that expected?
> 
> image

Hi @robert-bogos , that happens on hlx.page on all pages. You can see that here too https://main--cc--adobecom.hlx.page/adobecom.hlx.page/products/photoshop/remove-background.html
Assets are loaded from unity sharepoint hence this happens. It works fine on stage.adobe.com and adobe.com

**Review by sanjayms01 — APPROVED** [review]

**Comment by Ruchika4** [comment]
Merging this to stage for now. Testing can be only done once the prelude side of code is also ready. Once prelude is updated on stage will request @sigadamvenkata to do a thorough testing before getting it merged to production.

cc @sanjayms01 @dstrong23

**Pull request merged**